portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/Publications.testcase

```diff
@@ -563,6 +563,91 @@ definition {
 		Alert.viewErrorMessage(errorMessage = "Your request failed to complete.");
 	}
 
+	@priority = 3
+	test CanPublishAPublicationAfterAPageHasBeenDeleted {
```

Comment:
Name the test `CanPublishPublicationAfterPageHasBeenDeleted`. Usually we don't add articles (e.g. a, an, the) to  titles.